docs/content/browser.fsx

```diff
@@ -57,10 +58,10 @@ Now we need to tell React to render our view in the div placeholder we defined i
 by augumenting our program instance and passing the placeholder id:
 *)
 
-open Elmish.React
+open Elmish.HMR
```

Comment:
Let's not introduce HMR in this sample, @MangelMaxime has excellent intro in HRM repo.

Reply:
@et1975 I did because doesn't compile without it
Will try to get it to work without it

Reply:
Wonder what I did wrong last time, probably didn't properly install Elmish.React or didn't refresh the IDE.
Does work fine with it